src/message/messageActionSheet.js

```diff
@@ -58,7 +58,9 @@ const deleteMessage = async ({ auth, message, dispatch }) => {
   if (isAnOutboxMessage(message)) {
     dispatch(deleteOutboxMessage(message.timestamp));
   } else {
-    api.deleteMessage(auth, message.id);
+    api.deleteMessage(auth, message.id).catch((err: Error) => {
```

Comment:
A good follow up of this will be to make sure this message is translated based on user selected language (also for fail to edit message).

Reply:
Can you please tell me how I can do that? Th [/howto/translations.md](https://github.com/zulip/zulip-mobile/blob/master/docs/howto/translations.md) does not explain much.